README.md

```diff
@@ -148,16 +216,31 @@ Token token = Token.fromEnv();
 Conjur conjur = new Conjur(token);
 ```
 
-### Variable Operations
-
-Conjur variables store encrypted, access-controlled data. The most common thing a variable stores is a secret.
-A variable can have one or more (up to 20) secrets associated with it, and ordered in reverse chronological order.
+## Conjur Services Operation Examples
+This section provides some example operations on how invoke the Conjur Java API in your app. 
 
-You will typically add secrets to variables & retrieve secrets from variables in the following way:
+The most common use case is adding and retrieving a secret from Conjur. You can add secrets by invoking our API in the following way: 
 
 ```java
 conjur.variables().addSecret(VARIABLE_KEY, VARIABLE_VALUE);
-String retrievedSecret = conjur.variables().retrieveSecret(VARIABLE_KEY);
+```
+
+Alternatively, you can add a value to a secret in the Conjur CLI like so: 
+
+`conjur variable values add <NAME_OF_POLICY/VARIABLE> <VALUE_OF_SECRET>`. 
+
+Example: `conjur variable values add policyA/secret-var superSecret`
+
+_NOTE:_ Ensure that the secret exists in Conjur. To do so, you will need a policy that resembles the one supplied in the [Configuration](#configuration) section above.
```

Comment:
`Ensure that the secret exists in Conjur` -> `For a variable to be set, it first needs to be created by a policy otherwise this operation will fail.`